operators/text/string_ecmaregex_split.cc

```diff
@@ -15,40 +15,26 @@ KernelStringECMARegexSplitWithOffsets::KernelStringECMARegexSplitWithOffsets(con
   ignore_case_ = TryToGetAttributeWithDefault("ignore_case", false);
 }
 
-void KernelStringECMARegexSplitWithOffsets::Compute(OrtKernelContext* context) {
+void KernelStringECMARegexSplitWithOffsets::Compute(const ortc::Tensor<std::string>& input,
+                                                    std::string_view pattern,
```

Comment:
can these inputs be "const string_view&" ?

Reply:
string_view is read-only already.